cif_core.dic

```diff
@@ -15345,6 +15350,25 @@ save_audit_contact_author.id
     _type.source                  Related
     _type.container               Single
     _type.contents                Word
+    _method.purpose               Definition
+    _method.expression
+;
+    with aca as audit_contact_author
+    count = 0
+    target = 0
+    loop aa as audit_author {
+        if (aca.name == aa.name) {
+            count++
+            target = aa.id
+            }
+        }
+    if (count == 0 || count > 1) {
+        _enumeration.default = missing
+        }
+    else {
+        _enumeration.default = target
+    }
```

Comment:
```suggestion
    if (count == 1) {
        _enumeration.default = target
        }
    else {
        _enumeration.default = missing
    }
```

Reply:
Just a stylistic change for readability. Also avoids undefined behaviour if for some reason `count` becomes negative.